applications/blackhole/src/blackhole_socket_callback.erl

```diff
@@ -39,6 +39,12 @@ recv(_SessionPid, _SessionId, {'event', _Ignore, <<"subscribe">>, SubscriptionJO
     end,
     {'ok', Context1};
 
+recv(SessionPid, SessionId, {'event', _Ignore, <<"clear_subscriptions">>, _Data}, Context) ->
```

Comment:
@lazedo I was thinking about 'unsubscribe' of particular 'subscription' as opposite to 'clear_all' ('clear_subscriptions'). In the other words as a separate case . But if you think it is better to combine it, l'm all for.